priv/repo/migrations/20210921025336_change_image_scale_option_type.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Philomena.Repo.Migrations.ChangeImageScaleOptionType do
+  use Ecto.Migration
+  import Ecto.Query
```

Comment:
Not using Query, don't need it here

Reply:
Leftover from a change I ended up not making, sorrt